pytorch_lightning/trainer/data_loading.py

```diff
@@ -48,15 +48,30 @@ def init_train_dataloader(self, model):
         if EXIST_ITER_DATASET and isinstance(self.get_train_dataloader().dataset, IterableDataset):
             self.num_training_batches = float('inf')
         else:
+            if not 0. <= self.train_percent_check <= 1.:
+                raise ValueError(f"train_percent_check must lie in the range [0.0, 1.0], but got "
```

Comment:
pls use ` around functions and variables...

Reply:
Done